### PR TITLE
Continue French translation

### PR DESCRIPTION
**Pull request by fnetX**
as title. I hope I didn't miss anything important.

### CODE REVIEW

File: graphics/rcd/lang/fr_FR.txt [hunk 1]
```diff
@@ -31,42 +31,109 @@ strings {
...
+		ERROR_MESSAGE_HEADING_REMOVE: "Cela ne peut pas être supprimé …";
+		ERROR_MESSAGE_HEADING_COST:   "Coût prévu:";
+		ERROR_MESSAGE_HEADING_RETURN: "Retour prévu:";
+		ERROR_MESSAGE_UNREMOVABLE:    "Ce objet ne peut pas être supprimé ….";
+		ERROR_MESSAGE_UNOWNED_LAND:   "Vous n'êtes pas propriétaire du terrain!";
+		ERROR_MESSAGE_EXPENSIVE:      "Il vous faut au moins %1%.";
+		ERROR_MESSAGE_BAD_LOCATION:   "Emplacement inadéquat.";
+		ERROR_MESSAGE_SLOPE:          "Ne peut être construit sur une pente.";
+		ERROR_MESSAGE_UNDERGROUND:    "Ne peut être construit en souterrain.";
+		ERROR_MESSAGE_OCCUPIED:       "Something is in the way !";
```
Comment (Noordfrees) [inline graphics/rcd/lang/fr_FR.txt]: here

File: graphics/rcd/lang/fr_FR.txt [hunk 2]
```diff
@@ -31,42 +31,109 @@ strings {
...
 
+		// Staff window.
+		STAFF_MANAGEMENT_TITLE: "Personnel";
+		STAFF_SALARY:           "Salaire quotidien: %1%";
+		STAFF_DISMISS:          "Renvoyer";
+		STAFF_HIRE:             "Embaucher";
+		STAFF_NAME_MECHANIC:    "Mécanicien(ne) %d";
+		STAFF_NAME_HANDYMAN:    "Bricoleur %d";
+		STAFF_NAME_GUARD:       "Garde %d";
+		STAFF_NAME_ENTERTAINER: "Entertainer %d";
```
Comment (Noordfrees) [inline graphics/rcd/lang/fr_FR.txt]: below it's Animateurs so I'm not sure?

**Review by Noordfrees — COMMENTED** [review]
:+1:

I think there are two strings you might have overlooked (annotated inline). I don't speak French though so not sure about the second one.

**Comment by fnetX** [comment]
Thank you for spotting this. I fixed it, and also changed the gender neutral names to some which don't require listing both. At least I hope that "Mécano" is inclusive to non-male mechanics, too, I'm not perfectly sure about that one.

**Pull request merged**